Dockerfile.xpu

```diff
@@ -30,9 +30,19 @@ COPY requirements-common.txt /workspace/vllm/requirements-common.txt
 
 RUN --mount=type=cache,target=/root/.cache/pip \
     pip install --no-cache-dir \
-    --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/ \
     -r requirements-xpu.txt
 
+RUN git clone https://github.com/intel/pti-gpu && \
```

Comment:
why do we need to build it here?

do you have released version for this?

Reply:
This library doesn't have a released binary version. One alternative way is to download corresponding installation file and install it during docker build. But actually the build doesn't take much time. which way do you prefer?
```
wget https://registrationcenter-download.intel.com/akdlm/IRC_NAS/884eaa22-d56f-45dc-9a65-901f1c625f9e/l_intel-pti-dev_p_0.9.0.38_offline.sh 
sh ./l_intel-pti-dev_p_0.9.0.38_offline.sh
source <path-of-installation>/pti/latest/env/vars.sh

```

Reply:
@youkaichao  by the way, for installing through installation file, we need to force using silent mode and it's not so reliable. Better to use current compile way. Any more concerns?